cmd/minikube/cmd/image.go

```diff
@@ -310,6 +312,22 @@ var buildImageCmd = &cobra.Command{
 				// Otherwise, assume it's a tar
 			}
 		}
+		if runtime.GOOS == "windows" && strings.Contains(dockerFile, "\\") {
+			// if dockerFile is a DOS path, translate it into UNIX path
+			// but we need to ask for permission from user first
+			out.String("minikube detects that you are using DOS-style path %s. Should minikube convert it to UNIX-style by replacing all \\ to /? [y/n]", dockerFile)
+			var answer string
+			for {
+				fmt.Scan(&answer)
+				answer := strings.ToLower(answer)
+				if answer == "y" || answer == "yes" || answer == "n" || answer == "no" {
+					break
+				}
+			}
+			if answer == "y" || answer == "yes" {
```

Comment:
`answer` is always going to be empty here because you're creating `answer` with a new scope within the `for` loop on L322 because of the `:=`. If the `:` is removed it will update the value instead of creating a new one.